src/main/java/uk/ac/ebi/ena/readtools/common/reads/normalizers/htsjdk/SolexaQualityNormalizer.java

```diff
@@ -0,0 +1,12 @@
+package uk.ac.ebi.ena.readtools.common.reads.normalizers.htsjdk;
+
+import htsjdk.samtools.util.SolexaQualityConverter;
+import uk.ac.ebi.ena.readtools.common.reads.QualityNormalizer;
+
+public class SolexaQualityNormalizer implements QualityNormalizer {
+
+    @Override
+    public void normalize(byte[] qualities) {
+        SolexaQualityConverter.getSingleton().convertSolexaQualityCharsToPhredBinary(qualities);

```

Comment:
Looks ok. Just double checking. Notes to myself:

convertSolexaQualityCharsToPhredChars(byte[] solexaQuals)
---------------------------
Decodes an array of solexa quality ASCII chars into Phred ASCII space.

corresponds to I think:
---------------------------

https://en.wikipedia.org/wiki/FASTQ_format#Quality

Solexa/Illumina 1.0 format can encode a Solexa/Illumina quality score from -5 to 62 using ASCII 59 to 126 (although in raw read data Solexa scores from -5 to 40 only are expected)